grapher/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -748,23 +750,34 @@ export class ScatterPlotChart
         return this.domainDefault("y")
     }
 
+    readonly relativeAxisTickFormatter: (
```

Comment:
If it's a relative mode, the column type should be a percentage already and should have the correct format fn, right?

Reply:
Yeah you're right, seems like all these transforms should actually be done on the table. I think it might not be too much work to rewrite them. Will try it out